Library/Homebrew/cmd/commands.rb

```diff
@@ -27,18 +27,30 @@ def commands
   end
 
   def internal_commands
-    (HOMEBREW_LIBRARY_PATH/"cmd").children.select(&:file?).map { |f| f.basename(".rb").to_s }
+    find_internal_commands HOMEBREW_LIBRARY_PATH/"cmd"
   end
 
   def internal_development_commands
-    (HOMEBREW_LIBRARY_PATH/"dev-cmd").children.select(&:file?).map { |f| f.basename(".rb").to_s }
+    find_internal_commands HOMEBREW_LIBRARY_PATH/"dev-cmd"
```

Comment:
Note that `bin/brew` currently doesn't support internal developer-only commands. Not sure if we should take that into account here or extend `bin/brew` to support them. Thoughts?